File: kotlin/vci-client/src/main/java/io/mosip/vciclient/proof/jwt/JWTProof.kt
package io.mosip.vciclient.proof.jwt

import io.mosip.vciclient.constants.ProofType
import io.mosip.vciclient.proof.Proof

data class JWTProof(
    var jwt: String = "",
) : Proof {
    override val proofType: String = ProofType.JWT.value
}

File: js/src/proof/jwt/JWTProof.js
const Proof = require("../Proof");
const { ProofType } = require("../../constants/ProofType");

class JWTProof extends Proof {
    constructor(jwt) {
        super(ProofType.JWT);
        this.jwt = jwt;
      }
}

module.exports = JWTProof;
